source/NVDAObjects/__init__.py

```diff
@@ -961,7 +961,7 @@ def event_foreground(self):
 	def event_becomeNavigatorObject(self):
 		"""Called when this object becomes the navigator object.
 		"""
-		braille.handler.handleReviewMove()
+		braille.handler.handleReviewMove(shouldAutoTether=not eventHandler.lastReviewMoveDueToFollowing)

```

Comment:
This variable name isn't very clear if you don't have the surrounding context of this PR. Maybe choose another name?